AAP-39133: Added backup procedure with new compression variables (#3119)
ansible/aap-docs: downstream/modules/platform/ref-general-inventory-variables.adoc

Default = `rhel8`

> | `use_archive_compression` | | Controls whether the filesystem-related backup files will be compressed before being sent to the host to run the backup operation on a global level. If set to true, a `tar.gz` file is generated on each {PlatformNameShort} host, and then the gzip compression is used. If set to false, a simple tar file is generated. 

Reviewer: ```suggestion
| `use_archive_compression` | | Controls at the global level whether the filesystem-related backup files will be compressed before being sent to the host to run the backup operation. If set to `true`, a `tar.gz` file is generated on each {PlatformNameShort} host, and then the gzip compression is used. If set to `false`, a simple tar file is generated. 
```

Author: Updated the text.

Default = `true`

> This functionality can be controlled on a component level using the variable `<componentName>_use_archive_compression`. 

Reviewer: ```suggestion
This functionality can be controlled at the component level using the variable `<componentName>_use_archive_compression`. 
```

Author: Updated all instances.

> | `<componentName>_use_archive_compression` | a| Enables or disables archive compression on a component level by specifying the component in `<componentName>`.  

Reviewer: ```suggestion
| `<componentName>_use_archive_compression` | a| Enables or disables archive compression at the component level by specifying the component in `<componentName>`.  
```

Author: Updated the text.

For example:

* `automationgateway_use_archive_compression=true`

* `automationcontroller_use_archive_compression=true`

* `automationhub_use_archive_compression=true`

* `automationedacontroller_use_archive_compression=true`

Default = `true`

> This functionality can be controlled on a global level using the variable `use_archive_compression`.

Reviewer: ```suggestion
This functionality can be controlled at the global level using the `use_archive_compression` variable.
```

Author: Updated the text.

> | `use_db_compression` | | Controls whether the database-related backup files will be compressed before being sent to the host to run the backup operation on a global level. 

Reviewer: ```suggestion
| `use_db_compression` | | Controls at the global level whether the database-related backup files will be compressed before being sent to the host to run the backup operation. 
```

Author: Updated the text.

Default = `true`

> This functionality can be controlled on a component level using the variable `<componentName>_use_db_compression`. 

Reviewer: ```suggestion
This functionality can be controlled at the component level using the variable `<componentName>_use_db_compression`. 
```

Author: Updated the text.

| `<componentName>_use_db_compression` | a| Enables or disables archive compression on a database level by specifying the component in `<componentName>`.  

For example:

* `automationgateway_use_db_compression=true`

* `automationcontroller_use_db_compression=true`

* `automationhub_use_db_compression=true`

* `automationedacontroller_use_db_compression=true`

Default = `true`

> This functionality can be controlled on a global level using the variable `use_db_compression`.

Reviewer: ```suggestion
This functionality can be controlled at the global level using the `use_db_compression` variable.
```

Author: Updated the text.
